packages/mobile/src/components/audio/RNVideoAudioPlayer.tsx

```diff
@@ -100,7 +102,7 @@ export const RNVideoAudioPlayer = () => {
   const counter = useSelector(getCounter)
   // const repeatMode = useSelector(getRepeat)
```

Comment:
should we remove these?

Reply:
Curious if you think otherwise, but I think given this was just an experimental player for now; we should probably just delete this whole component. 
I was planning on doing a separate PR later this week to delete everything